libnm-core/nm-setting-tc-config.c

```diff
@@ -1246,7 +1366,9 @@ _qdiscs_to_variant (GPtrArray *qdiscs)
 	if (qdiscs) {
 		for (i = 0; i < qdiscs->len; i++) {
 			NMTCQdisc *qdisc = qdiscs->pdata[i];
+			gs_strfreev char **attrs = nm_tc_qdisc_get_attribute_names (qdisc);
 			GVariantBuilder qdisc_builder;
+			int y;
```

Comment:
the names are inside a `GHashTable`, which has `guint` type for the number of elements. We should not use a integer type that is smaller than the number of elements that we theoretically could have.